integration_tests/src/main/python/orc_write_test.py

```diff
@@ -281,3 +281,27 @@ def test_orc_write_column_name_with_dots(spark_tmp_path):
         lambda spark, path:  gen_df(spark, gens).coalesce(1).write.orc(path),
         lambda spark, path: spark.read.orc(path),
         data_path)
+
+
+# test case from:
+# https://github.com/apache/spark/blob/v3.4.0/sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/orc/OrcQuerySuite.scala#L371
+@ignore_order
+def test_orc_do_not_lowercase_columns(spark_tmp_path):
+    data_path = spark_tmp_path + "/ORC_DATA"
+    assert_gpu_and_cpu_writes_are_equal_collect(
+        # column is uppercase
+        lambda spark, path: spark.range(0, 1000).select(col("id").alias("Acol")).write.orc(path),
+        lambda spark, path: spark.read.orc(path),
+        data_path)
+    try:
+        # reading lowercase causes exception
+        with_cpu_session(lambda spark: spark.read.orc(data_path + "/CPU").schema["acol"])
+        assert False
+    except KeyError as e:
+        assert ("No StructField named acol" in str(e))
```

Comment:
We can keep this uniform with the GPU run below.
```suggestion
        assert "No StructField named acol" in str(e)
```

Reply:
Done